examples/mobilenet/README.md

```diff
@@ -0,0 +1,13 @@
+# MobileNet
+
+For evaluation, please go to [`examples/classification`](https://github.com/chainer/chainercv/tree/master/examples/classification).
```

Comment:
Please update `examples/classification` to contain the performance of this model. Also, please compare the performance of this implementation and that of the original implementation.

Reply:
OK

Reply:
I wrote.